frontend/model/state.js

```diff
@@ -74,8 +74,8 @@ sbp('sbp/selectors/register', {
     // Note: Update this function when renaming a Vuex module, or implementing a new one,
     // or adding new settings to the initialState above
     // Example:
-    // if (!state.notifications) {
-    //   state.notifications = []
+    // if (!state.preferences) {
+    //   state.preferences = {}
```

Comment:
Actually, I updated this because the previous comment was not correct.
```
if (!state.notifications) {
  state.notifications = []
}
```

But in this PR, the `state.notifications` is no longer an array, and it's an object. That's why I changed the comment. Or do you want me to change it to the following?
```
if (!state.notifications) {
  state.notifications = { items: [], status: {} }
}
```

Reply:
@Silver-IT sure, feel free to change it to that